src/main/java/echoserver/Presenter.java

```diff
@@ -5,18 +5,17 @@ public Presenter(Console console){
         this.console = console;
     }
 
-    public void showAwaitingInputMessage(){
-        this.console.outputToScreen("Awaiting Input\n");
-    }
-
-    public void displayInputFromSocket(ClientConnection clientConnection) {
-        this.console.outputToScreen("Client Input: " + clientConnection.readLastSavedInput() + "\n");
+    public void showAwaitingInputMessageOnPort(Integer port){
+        this.console.outputToScreen("Awaiting Input on Port: " + port + "\n");
     }
 
     public void showConnectionClosingMessage(){
         this.console.outputToScreen("Connection closing\n");
     }
 
+    public void showClientInput(String input){
+        this.console.outputToScreen("Client Input: " + input + "\n");
+    }
```

Comment:
I am wondering if we need to have a method for every possible message we log, or if there is a more generic logging interface we can introduce.